pop.css

```diff
@@ -0,0 +1,151 @@
+.pop-up-background {
+  height: 100vh;
+  width: 100vw;
+  position: fixed;
+  top: 0;
+  left: 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  backdrop-filter: blur(10px);
+}
+
+.pop-up-card {
+  border: 1px solid #dfe1e6;
+  border-radius: 4.26vw;
+  background-color: #fff;
+  width: 91.46vw;
+  overflow: auto;
```

Comment:
- [ ] As requested by the previous review you should stick to the design as much as possible. It will help you when you build the exact project required by the user in the future. If you see the design then there is the padding from the top and the button of the pop-up window and as I notice that there is no `X` buttons in the pop-up window so the user cannot close it. Please fix this issue and ask for another code reviewer.  Hint ` You can fix it by adding margin from top and button`
### Your design
![his](https://github.com/pulgosodog/Portfolio-template-1/assets/81727988/878ee606-4b8a-46c8-b7cc-215b97375386)

### Figma design
 
![figma](https://github.com/pulgosodog/Portfolio-template-1/assets/81727988/c5694974-67d7-44a0-beef-2b576591b190)